.github/workflows/docker.yml

```
@@ -0,0 +1,43 @@
+name: Push Docker images to Docker Hub 
+
+on:
+  push:
+    branches: [main, master]
```

You can simplify this to:

```suggestion
    branches: [main]
```